maps/src/MapBinner.cxx

```diff
@@ -394,15 +411,19 @@ MapBinner::BinTimestream(const G3Timestream &det, double weight,
 		pcoupling *= weight;
 
 		for (size_t i = 0; i < det.size(); i++) {
-			(*T)[detpointing[i]] += pcoupling.t*det[i];
-			(*Q)[detpointing[i]] += pcoupling.q*det[i];
-			(*U)[detpointing[i]] += pcoupling.u*det[i];
+			if (T)
+				(*T)[detpointing[i]] += pcoupling.t*det[i];
+			if (Q)
+				(*Q)[detpointing[i]] += pcoupling.q*det[i];
+			if (U)
+				(*U)[detpointing[i]] += pcoupling.u*det[i];
```

Comment:
Have you benchmarked this? Adding these branches could cause meaningful pessimization.